### PR TITLE
Migrate the code to node.js LTS version 16.x.x

### PR DESCRIPTION
**Pull request by jerome-benoit**
Signed-off-by: Jérôme Benoit <jerome.benoit@piment-noir.org>

## PR Checklist

- [x] Please add a description of your changes.
- [ ] We need your changes to come with unit tests in order to keep this project in quality and easy to maintain.
- [ ] If your changes contain any new feature please be sure to document it.
- [ ] Please add a link to the open issue or task that this pull request will resolve.
- [ ] Add a note to the changelog to indicate the change.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

If yes please indicate it and migration info into the CHANGELOG.md file.

---

### CODE REVIEW

**Comment by pioardi** [comment]
Looks like we have some problems with our github actions

**Comment by jerome-benoit** [comment]
> Looks like we have some problems with our github actions

Could you please: 

+ Put back a working npm token at the repo level for publishing on npm.js
+ Maybe have a look at the GitHub actions issue by creating a test PR. But it looks strange, it's a SSH key refusal so maybe it's more related to my account settings. 

?

I would like to make a 2.1.1 release with full current node.js LTS support.

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.2.0] - 2021-29-12
+
+### Breaking Changes
+
+- Support only NodeJS version 16.x.x
```
Comment (pioardi) [inline CHANGELOG.md:12]: Thanks jerome for this upgrade !
So if anyone use this version with 14.x or 12.x, poolifier is not going to work ? Is that accurate?

Comment (jerome-benoit) [reply CHANGELOG.md:12]: For cluster pool, yes.

**Review by pioardi — APPROVED** [review]
Thanks for your effort to work on that

**Pull request merged**